### ISSUE TITLE
Questions about SISDRLoss

### ISSUE THREAD

**Issue by PrShi113** [csteinmetz1/auraloss#41]
Hi! Thank you for your useful loss functions. But could you explain the value of SISDRLoss? I try to train a model for the speech separation task.
Is it normal that the value of SISDRLoss is always negative?
And how can I determine whether the model is converging? Should the value of SISDRLoss converge to 0, or keep a large value?

**Comment by csteinmetz1** [comment]
Hi @PrShi113,

Normally a high (positive) SI-SDR means the model is performing well. However, since we want to use the SI-SDR as a loss function and by default PyTorch optimizer will attempt to minimize the loss, in `auraloss` the `SISDRLoss` will automatically be negated. This means that the more negative the value from the loss, the better the model is doing. This loss is in theory unbounded and as the model is converging it should be going more negative. I hope this answers your question. 

If you have any other doubts or issues feel free to open another issue.
